### PR TITLE
Update roadmap for v4.5.0 release

### PR DESCRIPTION
**Pull request by calvin-lau-sig7**
Updates the roadmap that moves up a couple of items, also adds things from v4.5.0 release.  Aim is to publish this alongside v4.5.0 release expected on 31 January 2022

### CODE REVIEW

File: src/community/roadmap/index.md [hunk 1]
```diff
@@ -40,9 +41,6 @@ We're working to:
 We're getting ready to:
```
Comment (calvin-lau-sig7) [inline src/community/roadmap/index.md:41]: Any upcoming stuff we want to add here

Comment (stevenjmesser) [reply src/community/roadmap/index.md:41]: It'd be great to mention the guidance on adding findings from research that @christopherthomasdesign is looking at.

Comment (stevenjmesser) [reply src/community/roadmap/index.md:41]: Think we should mention that we're planning the next priorities too.

Comment (stevenjmesser) [reply src/community/roadmap/index.md:42]: Have added these in a commit!

**Review by stevenjmesser — APPROVED** [review]
Everything looks good!

File: src/community/roadmap/index.md (hunk 1 above)
Comment (calvin-lau-sig7) [reply src/community/roadmap/index.md:42]: Looks good, thanks for adding and approving!

**Pull request merged**